test/blackbox/run_blackbox_tests/test_optimal_pmtu.c

```diff
@@ -57,42 +61,6 @@ bool ping_channel_enable_07 = false;
 struct sync_flag test_pmtu_nut_closed = {.mutex  = PTHREAD_MUTEX_INITIALIZER, .cond = PTHREAD_COND_INITIALIZER};
 static netns_state_t *test_pmtu_state;
 
-static int setup_test(void **state) {
-	netns_create_topology(test_pmtu_state);
-	fprintf(stderr, "\nCreated topology\n");
-
-	test_pmtu_relay_running = true;
-	test_pmtu_peer_running = true;
-	test_pmtu_nut_running = true;
-	ping_channel_enable_07 = false;
-	memset(node_pmtu, 2, sizeof(node_pmtu[0]));
```

Comment:
Ah, I see a mistake here, maybe that is why you changed the code to use pointers? It should have been:
`memset(node_pmtu, 0, sizeof(node_pmtu))`